sycl/cmake/modules/AddSYCLUnitTest.cmake

```diff
@@ -67,6 +67,10 @@ macro(add_sycl_unittest test_dirname link_variant)
       ${SYCL_SOURCE_DIR}/source/
       ${SYCL_SOURCE_DIR}/unittests/
     )
+  check_cxx_compiler_flag(-Wno-ignored-attributes HAS_WNO_IGNORED_ATTRS)
+  if (HAS_WNO_IGNORED_ATTRS)
+    target_compile_options(${test_dirname} PRIVATE -Wno-ignored-attributes)
+  endif()
```

Comment:
This needs a comment about why we are doing this instead of fixing the warning.

Reply:
I reverted CMake changes.